### PR TITLE
Expose Component and helpers as named imports for ES6 Module syntax

### LINKED ISSUE #84: Expose Helpers and Component on main Export

### PR DESCRIPTION
**Pull request by BigAB**
Resolves #84

### CODE REVIEW

File: react-view-model.js [hunk 1]
```diff
@@ -55,3 +58,9 @@ module.exports = namespace.reactViewModel = function reactViewModel(displayName,
 
 	return App;
 };
+
+// Expose Component and helpers to named imports
+module.exports.Component = Component;
+module.exports.promise = promise;
```
Comment (christopherjbaker) [inline react-view-model.js]: I think `promise` should be `Promise`, as it is a constructor.

File: react-view-model.js [hunk 2]
```diff
@@ -55,3 +58,9 @@ module.exports = namespace.reactViewModel = function reactViewModel(displayName,
...
+module.exports.autoBindMethods = autoBindMethods;
```
Comment (christopherjbaker) [inline react-view-model.js]: should `autoBindMethods` be `autobindMethods` to match the filename?

Comment (BigAB) [reply react-view-model.js]: Good question. I'll change it one way or the other, but which is correct?

Is it `auto-bind-methods` and `autoBindMethods`?
or `autobind-methods` and `autobindMethods`?

File: react-view-model.js (hunk 1 above)
Comment (BigAB) [reply react-view-model.js]: Also a good point, but then we would be "masking" the native `Promise` in the file this was imported into. How about ...?
`PromiseHelper`?
`PromiseMap`?
`ObservablePromise`?
`PromiseObs`?

...any better ideas?

File: react-view-model.js (hunk 2 above)
Comment (christopherjbaker) [reply react-view-model.js]: i like autobind-methods. If you did it with spaces, you'd do "autobind methods", not "auto bind methods", as "auto" is not its own word.

File: react-view-model.js (hunk 1 above)
Comment (christopherjbaker) [reply react-view-model.js]: hmm. Also a good point. I would go with `ObservablePromise` or `PromiseHelper`.

**Pull request merged**